app/controllers/questions_controller.rb

```diff
@@ -26,6 +26,16 @@ def destroy
     flash[:notice] = "Question successfully deleted" 
   end
 
+  def clear_votes
+    question = Question.find(params[:id])
+    if question.choices.map{ |choice| choice.votes.destroy_all }
+      redirect_to question_path(question)
+      flash[:notice] = "Votes successfully cleared"
+    else
+      flash[:alert] = "Error clearing votes"
```

Comment:
I'll make a card to only show delete votes button if there is votes. low priority